k4EDM4hep2LcioConv/include/k4EDM4hep2LcioConv/MappingUtils.h

```diff
@@ -0,0 +1,229 @@
+#ifndef K4EDM4HEP2LCIOCONV_MAPPINGUTILS_H
+#define K4EDM4HEP2LCIOCONV_MAPPINGUTILS_H
+
+#include <optional>
+#include <algorithm>
+#include <vector>
+#include <tuple>
+#include <unordered_map>
+#include <type_traits>
+
+#if __has_include("experimental/type_traits.h")
```

Comment:
```suggestion
#if __has_include("experimental/type_traits")
```
But why do you have `#include <type_traits>`, and then again `#include <experimental/type_traits>`?

Reply:
Ok, experimental/type_traits still have type_traits that are experimental.